src/plugins/intel_gpu/src/plugin/ops/eye.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/op/eye.hpp"
+
+#include <memory>
+
+#include "intel_gpu/plugin/common_utils.hpp"
+#include "intel_gpu/plugin/program.hpp"
+#include "intel_gpu/primitives/eye.hpp"
+
+namespace ov {
+namespace runtime {
+namespace intel_gpu {
+
+namespace {
+
+static void CreateEyeOp(Program& p, const std::shared_ptr<ngraph::op::v9::Eye>& op) {
+    p.ValidateInputs(op, {3, 4});
+
+    const InferenceEngine::SizeVector& output_shapes = op->get_output_shape(0);
+    auto os_sz = output_shapes.size();
+    /*static constexpr*/ size_t dim_size = 4;
```

Comment:
why have you commented static constexpr?

Reply:
return back